### ISSUE TITLE
Option to show Button Hitboxes

### ISSUE THREAD

**Issue by Unspeakable-Boy** [absolllute/Mega-Hack-Pro-Future#1067]
Pretty streight forward; hitbox option to show button hitboxes. Could possibly find hidden buttons with it.

**Comment by NewAlexGamer** [comment]
you mean this trigger orb?
![image](https://user-images.githubusercontent.com/86263500/191520700-dc38772e-391c-4df3-abbc-c8242d57208c.png)

**Comment by Silas123Go** [comment]
> you mean this trigger orb? if yes it already shows hitboxes as you can see
> ![image](https://user-images.githubusercontent.com/86263500/191520700-dc38772e-391c-4df3-abbc-c8242d57208c.png)

He means buttons as in like, the icon kit button, profile button, etc. (i think)

**Comment by NewAlexGamer** [comment]
oopsie

**Comment by Unspeakable-Boy** [comment]
Yeah, I mean like any clickable thing/button IN THE GAME. Not just editor.

**Comment by absoIute** [comment]
This feels like more of a debug thing than an something that'd suit Mega Hack. cocos-explorer does this if you hold shift over an object!
